src/app/settings/settings.component.ts

```diff
@@ -231,4 +231,13 @@ export class SettingsComponent extends RouterEnter {
     return this.authService.logout();
   }
 
+  setCursorPosition(event) {
```

Comment:
I didn't get what is this function doing? what does `return true;` / `return false;` mean?